### PR TITLE
Add MultiAZ parameter in rds_instances

### PR DESCRIPTION
**Pull request by eduherraiz**
Initial work trying to solve: #4325 
I can say the parameter in my actual case was true by default (aurora postgresql 9.6.6).

### CODE REVIEW

File: aws/resource_aws_rds_cluster_instance.go [hunk 1]
```diff
@@ -196,6 +196,12 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
 				ValidateFunc: validateArn,
 			},
 
+			"multiaz": {
```
Comment (bflad) [inline aws/resource_aws_rds_cluster_instance.go]: A few things with this new attribute:
* Can you please document this new argument in `website/docs/r/rds_cluster_instance.html.markdown`?
* Can you please adjust this to be named `multi_az` to match the `aws_db_instance` resource? https://www.terraform.io/docs/providers/aws/r/db_instance.html#multi_az
* Can you please add a new acceptance test in `aws/resource_aws_rds_cluster_instance_test.go` with this new argument set to `false`? Copy paste of an existing test and test configuration is a great way to get started 👍

**Comment by eduherraiz** [comment]
**TLDR**: My mistake, The MultiAZ in Aurora is only a flag saying we have two instances in different zones.

Doing more tests I can see I forgot something.
Trying to enable the multi_az for a Aurora Postgresql was created with a false value I get the error:
`InvalidParameterCombination: VPC Multi-AZ DB Instances are not available for engine: aurora-postgresql`

I can say now it's only my mistake understanding the module. 
On mark the "MultiAZ" option on the web console create 2 separated instances in the cluster. 
It's no used by the API, when create the instance.
